go.mod

```diff
@@ -1,6 +1,6 @@
 module kpt.dev/configsync
 
-go 1.23.0
+go 1.24.3
```

Comment:
I'd suggest updating just the image version initially, otherwise CI may break in the interim (until the CI environment is updated with the new Go version)

Reply:
reverted